src/System.Security.Cryptography.Xml/tests/DSAKeyValueTest.cs

```diff
@@ -88,7 +88,7 @@ public void GetXml_SameDsa()
             }
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/corefx/issues/16779")]
```

Comment:
@bartonjs I'm not converting to ActiveIssue while there is some active development happening and until this is fixed: https://github.com/dotnet/corefx/issues/16686 - in the issue I meant that the exactly the same test build and run against full .net framework is passing. possibly corefx test is failing when run on desktop or linux is using wrong assemblies - I'll investigate that when I get to this failure. I plan on fixing all of them before publishing the package.